spec/cli_v1_commands.yaml

```diff
@@ -673,7 +690,7 @@ commands:
         code: supabase init
         response: Finished supabase init.
       - id: from-workdir
-        name: Initialize from an existing directory
```

Comment:
Same as previous spelling. Was it changed because of my local settings?

Reply:
Good catch. I think it was off sync with cli's docs/templates/examples.yaml. We should update it in cli but can be done in a separate PR